### PR TITLE
Upgrade to TF 1.15

### PR DESCRIPTION
**Pull request by gogasca**
Upgrade sample to TF 1.15

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/466)

### CODE REVIEW

**Comment by gogasca** [comment]
Thanks @alecglassford updated notebook with suggestions

File: notebooks/tensorflow/getting-started-keras.ipynb [hunk 1]
```diff
@@ -1375,32 +1399,11 @@
...
-        "In this example, the training code uses a [TensorBoard\n",
-        "callback](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard),\n",
-        "which [creates TensorFlow `Summary`\n",
-        "`Event`s](https://www.tensorflow.org/api_docs/python/tf/summary/FileWriter#add_summary)\n",
-        "during training. AI Platform uses these events to track the metric you want to\n",
-        "optimize. Learn more about [hyperparameter tuning in\n",
-        "AI Platform Training](https://cloud.google.com/ml-engine/docs/tensorflow/hyperparameter-tuning-overview)."
-      ]
-    },
     {
```
Comment (alecglassford) [inline notebooks/tensorflow/getting-started-keras.ipynb]: I think this cell is partially duplicated? See https://colab.research.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/e88c0b6a7e6ed7bf4168a9de62cf9c42c4256c72/notebooks/tensorflow/getting-started-keras.ipynb#scrollTo=J2WGqwAzc3xM

I would delete lines 1399 to 1419.

Comment (alecglassford) [reply notebooks/tensorflow/getting-started-keras.ipynb]: duplicate-cell

Comment (gogasca) [reply notebooks/tensorflow/getting-started-keras.ipynb]: Did modification, removed cell. Thanks @alecglassford

**Pull request merged**